### PR TITLE
:speech_balloon: Updated Readme

### PR DESCRIPTION
**Pull request by NalinPlad**
Changed header to HTML so that the content could be centered. I also slightly reduced the size of the image. One of the badges was broken so I replaced it as well.

### CODE REVIEW

**Review by lostdesign — CHANGES_REQUESTED** [review]
Can you remove the div and only use the img tags in the following order instead?

```html
<img width=90% src="https://user-images.githubusercontent.com/5164617/132354376-44ba85f3-bb21-448e-8c12-4e0fe9af2543.png" alt="Promotional Image"/>
<br>
<img alt="GitHub release (latest by date)" src="https://img.shields.io/github/v/release/lostdesign/linked">
<img alt="Project downloads" src="https://img.shields.io/github/downloads/lostdesign/linked/total"/>
<img alt="Discord" src="https://img.shields.io/discord/884747974859702352?label=discord"/>
<img alt="GitHub issues" src="https://img.shields.io/github/issues/lostdesign/linked">
```

File: README.md [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
+    <img alt="GitHub issues" src="https://img.shields.io/github/issues/lostdesign/linked">
+  </h1>
+  <img width=90% src="https://user-images.githubusercontent.com/5164617/132354376-44ba85f3-bb21-448e-8c12-4e0fe9af2543.png" alt="Promotional Image"/>
+</div>
 
 # Daily journaling without distraction
 An easy, distraction-free way to record your thoughts, declutter your mind and keep the things you want to remember. We all only have so much room in our brains for things. Let Linked be the place to put something you want to remember and let you free up space for deeper thinking.
 
-**[You can download linked here for free!](https://github.com/lostdesign/linked/releases/latest)**
+**[You can download it here for free!](https://github.com/lostdesign/linked/releases/latest)**
```
Comment (lostdesign) [inline README.md:15]: I think it should stay linked in this case.

**Review by lostdesign — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (NalinPlad) [reply README.md:15]: Sounds good

**Comment by NalinPlad** [comment]
The div is neccesary to keep the images centered

**Comment by lostdesign** [comment]
> The div is neccesary to keep the images centered

Not really a fan of it being centered tbh

**Comment by NalinPlad** [comment]
I changed it, now the badges are on the left

**Comment by NalinPlad** [comment]
It's a minor change and not much reason to keep it up when there are more important pull requests to review. I'm not demanding for anyone to do anything

**Pull request closed without merging**